lib/ops/opsservice/service.go

```diff
@@ -165,6 +165,11 @@ type Operator struct {
 
 	// FieldLogger allows this operator to log messages
 	log.FieldLogger
+
+	// cachedProvisioningToken holds an in memory cache of which token is the cluster provisioning token.
+	cachedProvisioningToken string
+	// cachedProvisioningTokenMutex provides a mutex on use of the cached provisioning token.
+	cachedProvisioningTokenMutex sync.RWMutex
```

Comment:
nit: I think it's more common to declare the mutex first and then the fields it is supposed to protect